ballerina/tests/from_json_string_test.bal

```diff
@@ -1595,3 +1692,26 @@ function testComplexTypeAsUnionMemberAsExpTypeNegative() {
     test:assertTrue(t2 is error);
     test:assertEquals((<error>t2).message(), "unsupported type '(map<anydata>|int)'");
 }
+
+@test:Config
+isolated function testConvertNonStringValueNegative() {
+    string|Error err1 = parseString("null");
```

Comment:
Shall we a test for ()

Reply:
Added https://github.com/ballerina-platform/module-ballerina-data.jsondata/pull/4/commits/f22736abe625915c6de5af9aa540a5c88b345bc0